system_metrics_collector/src/topic_statistics_collector/constants.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef TOPIC_STATISTICS_COLLECTOR__CONSTANTS_HPP_
+#define TOPIC_STATISTICS_COLLECTOR__CONSTANTS_HPP_
+
+#include <string>
+
+namespace topic_statistics_collector
+{
+
+namespace topic_statistics_constants
+{
+constexpr const char kMsgAgeStatName[] = "message_age";
+constexpr const char kMsgPeriodStatName[] = "message_period";
+constexpr const char kMillisecondUnitName[] = "ms";
```

Comment:
constant strings need to be defined in the matching cpp file.

Reply:
This pattern also exists in this package [elsewhere](https://github.com/ros-tooling/system_metrics_collector/blob/master/system_metrics_collector/src/system_metrics_collector/constants.hpp). I'll address this comment everywhere in a separate PR to keep the diff short here.

Reply:
+1 - let's make sure this gets fixed, as this could cause "invisible issues" as we don't have a CI attempting to compile in release, for instance.

Reply:
I'll file an issue.

Reply:
Filed https://github.com/ros-tooling/system_metrics_collector/issues/116